ppb_vector/vector2.py

```diff
@@ -213,6 +213,9 @@ def __add__(self: Vector, other: VectorLike) -> Vector:
             return NotImplemented
         return rtype(self.x + other_x, self.y + other_y)
 
+    def __radd__(self: Vector, other: VectorLike) -> Vector:
+        return self.__add__(other)
```

Comment:
```suggestion
        return self + other
```

Reply:
I'm not sure why this? Other than some sense you shouldn't call dunders directly?